src/features/peers/redux/reducer.ts

```diff
@@ -11,7 +11,7 @@ export const peers: Reducer<string[], UpdatePeersAction> = (
         case UPDATE_PEERS: {
             return payload
         }
-        case RESET_APP_STATE: {
+        case STOCK_CHANGE: {
             return ['']
```

Comment:
I'm not sure if this is wrong, but it looks a little odd -- a string array with a single item of an empty string. Is that what you want or would an empty array be more appropriate?

Reply:
Yes I saw this and figured I'd just have to put an empty object first but it seems to work regardless.

I'll change to empty object preceeding state but will also look into why this might work.

Reply:
.....yes.

Changed to an empty array.